middleware/auth.js

```diff
@@ -12,3 +12,19 @@ exports.authorize = function (req, res, next) {
             res.status(401).send('Unauthorized');
         });
 };
+exports.adminAuthorize = function (req, res, next) {
```

Comment:
Could we add a few tests to either test/auth_test.js or a new test/admin_auth_tests.js to check the admin authorization with requests?